cpp/roaring64map.hh

```diff
@@ -144,10 +144,10 @@ class Roaring64Map{
      * Check if value x is present
      */
     bool contains(uint32_t x) const {
-        return roarings.at(0).contains(x);
+        return roarings.count(0) == 0 ? false : roarings.at(0).contains(x);
```

Comment:
rather big deal here, at() will throw an exception if we don't check count() first